iml-gui/crate/src/components/dropdown.rs

```diff
@@ -0,0 +1,57 @@
+use crate::{components::Placement, generated::css_classes::C};
```

Comment:
copyright?

Reply:
Not yet. Will add to all files before publishing first version.